CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [2.1.1] - 2023-08-22
+
+### Changed
+- Updated to require notebook < 7.0
```

Comment:
would it make sense to include more info, like from the readme? 

> Jupyter released a major version upgrade for notebooks that has breaking changes. notebook >=7.0 has major changes to the way extensions are handled, detailed in their documentation: https://jupyter-notebook.readthedocs.io/en/stable/migrate_to_notebook7.html. This is causing the civis-jupyter-notebook build to fail and until we can update the extensions we use, I'm pinning the notebook version to <7.0.
Also fixed security vulnerability around not using timeout with requests. Platform has a 60 second timeout so I reflected that here.


```suggestion
- Updated to require notebook < 7.0, as [jupiter-notebook 7.0](https://jupyter-notebook.readthedocs.io/en/stable/migrate_to_notebook7.html) introduced breaking changes. 
- Fixed security vulnerability around not using timeout with requests.
```

Reply:
Yeah, couldn't hurt!